docs/index.md

```diff
@@ -17,6 +17,7 @@
 
 Zulip is an online platform that we use for the CLAM Community. It is similar to Slack or Discord, but has the following benefits:
 
+- **Viewable without an account**: Zulip allows members of the community who don't have an account to follow and benefit from community discussions.
```

Comment:
This is something that we actually need to discuss and make a decision on - which channels should be fully public?